server/base/src/main/java/org/apache/accumulo/server/AbstractServer.java

```diff
@@ -87,6 +93,24 @@ public void runServer() throws Exception {
     }
   }
 
+  @Override
+  public void registerMetrics(MeterRegistry registry) {
+    lowMemoryMetricGuage =
+        Gauge
+            .builder(METRICS_APP_PREFIX + applicationName + "." + hostname + "."
```

Comment:
I'm not super familiar with Micrometer, but the way this is currently implemented, wouldn't
the metrics be created in the following structure?
`accumulo.app.app1.server1.detected.low.memory = 1|0` 
`accumulo.app.app1.server2.detected.low.memory = 1|0`
`accumulo.app.app2.server1.detected.low.memory = 1|0`

I'm pretty sure these would all be considered unique metrics.

Does Micrometer support tags or metric labels? 
If the application name and host name were added as tags on the metric, then the metric would look like
`accumulo.app.detected.low.memory{"application Name": "app1", "hostname": server1} = 1|0` 

This drops the unique metric definition down to `accumulo.app.detected.low.memory` and allows sorting and filtering based on `application name` or `hostname`. 

Alerting still works by firing on `accumulo.app.detected.low.memory =1` but better granularity is allowed on the visualization and alerting ends.

Reply:
There are tags - I'm trying to confirm that they are being set correctly as I look at Dave's other comments.

Reply:
I agree, I think we want something like `accumulo.server.detected.low.memory` and the application name should be in a tag.

Reply:
Leveraged tags in 674a9da956